src/test/java/seedu/address/logic/commands/AddAppCommandTest.java

```diff
@@ -160,7 +160,7 @@ public boolean hasPersonWithNric(Nric nric) {
 
         @Override
         public Person getPersonWithNric(Nric nric) {
-            throw new AssertionError("This method should not be called.");
+            return null;
         }
```

Comment:
Is this method duplicated? Seems to also be over-riden below in lines 271-279.

Reply:
I think there's 2 versions of the model stub

Reply:
Oh yeah i changed this when i was doing the getPersonWithNric in AppCommand, will change it back, thanks for the review!